releasenotes/notes/Adding-BackendV2-support-to-schedule-of-qiskit.compiler.scheduler-a0366d3397c73af0.yaml

```
@@ -0,0 +1,20 @@
+---
+fixes:
+  - |
+    Enables scheduling using BackendV2.
+      Example usage::
+      #import a fake backend which is a sub-class of BackendV2.
+      from qiskit.providers.fake_provider import FakePerth
+      from qiskit.compiler.scheduler import schedule
+      from qiskit.circuit import QuantumCircuit
+      qc=QuantumCircuit(1, 1)
+      qc.x(0)
+      qc.measure(0,0)
+      sched=schedule(circuits=qc, backend=FakePerth())
+
+    Since, BackendV2 was not supported by schedule method in qiskit.compiler.scheduler, this caused
+    schedule method to error out when the backend is supplied with BackendV2. Refer to
+    `#10837 <https://github.com/Qiskit/qiskit/issues/10837>` for more information.
+
+
+
```

```suggestion
features:
  - |
    Enables circuit to pulse scheduling using BackendV2.
    
    .. code-block:: python
    
      # import a fake backend which is a sub-class of BackendV2.
      from qiskit.providers.fake_provider import FakePerth
      from qiskit.compiler.scheduler import schedule
      from qiskit.circuit import QuantumCircuit
      
      qc = QuantumCircuit(1, 1)
      qc.x(0)
      qc.measure(0,0)
      sched = schedule(circuits=qc, backend=FakePerth())

    Since, BackendV2 was not supported by the :func:`~qiskit.compiler.schedule` function, this caused
    schedule method to error out when the :code:`backend` argument is supplied with :class:`.BackendV2`. 
    Refer to `#10837 <https://github.com/Qiskit/qiskit/issues/10837>` for more information.

```

Cleanup. I think this should be classified into new feature, because the PR enables new object input which has not been supported previously.